app/f0_serialcom/FreeRTOSConfig.h

```diff
@@ -0,0 +1,88 @@
+/*
```

Comment:
Ok, cool, that's why I didn't comment on the style. Figured this was the case from the license comment block.